### PR TITLE
comply with dplyr 1.0.0 deprecation of add argument for group_by

### PR DESCRIPTION
**Pull request by davidtedfordholt**
This PR includes changes to three files, two in order to use the `.add` argument instead of the `add` argument  and one in order to allow the `group_by.tbl_ts` function to follow the same convention. It includes code to ensure that current uses of the `add` argument will continue to work, but will give a warning message, pointing the user to the dplyr deprecation.

### CODE REVIEW

File: R/dplyr-verbs.R [hunk 1]
```diff
@@ -199,11 +199,17 @@ summarise.tbl_ts <- function(.data, ...) {
...
 #' @export
-group_by.tbl_ts <- function(.data, ..., add = FALSE,
-                            .drop = group_by_drop_default(.data)) {
+group_by.tbl_ts <- function(.data, ..., .add = FALSE,
+                            .drop = group_by_drop_default(.data),
+                            add = FALSE) {
+  if (!missing(add)) {
+    warning("The `add` argument of `group_by()` is deprecated as of dplyr 1.0.0. Please use `.add`.")
+    .add <- add
+  }
```
Comment (earowang) [inline R/dplyr-verbs.R]: Can you remove the warning? The `NextMethod()` in the later part of the function should trigger a warning from dplyr.

**Review by earowang — COMMENTED** [review]
Can you bump the dependency to `dplyr (>=1.0.0)` and list it in the remote field in the DESCRIPTION?

File: R/dplyr-verbs.R (hunk 1 above)
Comment (davidtedfordholt) [reply R/dplyr-verbs.R]: Dependency and remote entry added, and removed the warning.

**Comment by earowang** [comment]
Thanks.

**Pull request merged**